cmd/plasmacli/subcmd/keys/export.go

```diff
@@ -0,0 +1,55 @@
+package keys
+
+import (
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/cmd/plasmacli/store"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+// ExportCmd returns the keys export command
+func ExportCmd() *cobra.Command {
+	return exportcmd
+}
+
+var exportcmd = &cobra.Command{
+	Use:   "export <name> <location>",
+	Short: "Export a private key",
+	Long: `Exports a private key to a specified location (must be absolute path).
+Prints the address. 
```

Comment:
`Prints the address` this functionality is missing, I have no preference on if it is necessary or not